README.md

```diff
@@ -1,7 +1,117 @@
-PyYAML
-======
+PyYAML-ft
+=========
+
+A full-featured YAML processing framework for Python with support for free-threading
+
+## Why a fork?
+
+[PEP 703](https://peps.python.org/pep-0703/) introduced free-threaded Python as a
+separate build of CPython 3.13. Thread-safety issues that might have otherwise gone
+unnoticed are now much easier to trigger because of the absence of protection from
+the GIL. Also, because the free-threaded build is ABI-incompatible, extension
+modules need to be separate, free-threaded wheels and declare support for
+it.
+
+The PyYAML maintainers
+[decided to not port PyYAML to the free-threaded build](https://github.com/yaml/pyyaml/pull/830#issuecomment-2342475334)
```

Comment:
FTR, this is inaccurate: https://github.com/yaml/pyyaml/issues/856#issuecomment-3063058860